File: Basic Coding Problems/01 Fundamentals/014 Write a program to enter the base and height of a triangle and find its area/C++/triangle_area.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    float base, height, area;

    cout << "Enter the base of the triangle: ";
    cin >> base;

    cout << "Enter the height of the triangle: ";
    cin >> height;

    area = 0.5 * base * height;

    cout << "The area of the triangle is: " << fixed << setprecision(2) << area << endl;

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/014 Write a program to enter the base and height of a triangle and find its area/Java/triangle_area.java
import java.util.Scanner;

public class triangle_area {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the base of the triangle: ");
        float base = scanner.nextFloat();

        System.out.print("Enter the height of the triangle: ");
        float height = scanner.nextFloat();

        float area = (float) 0.5 * base * height;

        System.out.printf("The area of the triangle is: %.2f\n", area);
    }
}